### DISCUSSION TITLE
Hello, does Github recognises online degree programs like BSc by Birla Institute of Technology & Science, Pilani on coursera ?

### DISCUSSION THREAD

**Discussion by ayushagarwal27** [community/community#50151] [GitHub Education]
I have applied with college provided online e-mail id, however github is not able to recognise the said id.

**Comment by samus-aran** [comment]
Hi @ayushCode27 

We're happy you're here 👋 

To be eligible for GitHub Global Campus, including GitHub Student Developer Pack and other benefits, you must:

- **Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution**
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
- Be at least 13 years old

For more information, read [Requirements](https://docs.github.com/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#requirements) in the GitHub Docs.

You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by ayushagarwal27** [reply]
![Screenshot 2023-03-17 at 12 38 17 PM](https://user-images.githubusercontent.com/67559031/225836425-3f5d460c-ae7d-486b-8a1b-37e3fca68a5b.png)

This is the message I received. Student ID, uploaded clearly states institution name and mode: Online.
